clang-tools-extra/clang-tidy/bugprone/SuspiciousPointerArithmeticsUsingSizeofCheck.h

```diff
@@ -0,0 +1,30 @@
+//===--- SuspiciousPointerArithmeticsUsingSizeofCheck.h - clang-tidy -*- C++ -*-===//
```

Comment:
Length.

Reply:
This is not fixable without removing the `-*- C++ -*-` part, but that also should be there, because emacs users.